dist/t/spec/spec_helper.rb

```diff
@@ -11,8 +11,33 @@
   config.disable_monkey_patching!
 
   # Run specs in random order to surface order dependencies
-  config.order = :random
+  ##config.order = :random  # we will follow one workflow for now...
```

Comment:
Don't comment things out. Remove them instead.

Though why don't we want to use random test order?